### PR TITLE
Fulcrum umi

### PR DESCRIPTION
**Pull request by bryce-turner**
This adds a workflow for processing UMIs. Might need adjustments/improvements.

### CODE REVIEW

File: modules/dna_alignment/fgbio_umi.jst [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    - {{ temp_dir }}/{{ sample.rgsm }}_UMIcorr_bwa.bam
+  cpus: 10
+  walltime: "4:00:00"
+  cmd: |
+    set -eu
+    set -o pipefail
+
+    module load {{ constants.tools.fgbio.module }}
+
+    KNOWN_UMI=/scratch/bturner/stamina_fastqs/Twist_beta_known_umi.txt
```
Comment (bryce-turner) [inline modules/dna_alignment/fgbio_umi.jst]: Need to fix this before production

**Comment by bryce-turner** [comment]
Ready for review!

File: modules/annotation/vep.jst [hunk 2]
```diff
@@ -112,4 +112,5 @@
 
     bcftools index --threads 4 --force --tbi "{{ final_vcf_prefix }}.vep.pick.vcf.gz"
 
+
```
Comment (PedalheadPHX) [inline modules/annotation/vep.jst]: @bryce-turner won't this space cause a bunch of restarts?

**Review by PedalheadPHX — COMMENTED** [review]
@bryce-turner Just need to resolve the one issue with a space and if that would cause restarts if not corrected

**Review by PedalheadPHX — APPROVED** [review]
Good for integration with development now

**Pull request merged**